web/src/components/users/RootPasswordPopup.jsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) [2022] SUSE LLC
```

Comment:
2023

Reply:
It's actually from 2022, but let's update the dates to 2022-2023